gradle/plugins/src/main/kotlin/com.hedera.gradle.spotless-markdown.gradle.kts

```
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id("com.hedera.gradle.spotless") }
+
+spotless {
+    flexmark {
+        target("**/*.md", ".gitignore")
```

The `.gitignore` file should not be formatted using a markdown linter.

```suggestion
        target("**/*.md")
```